Trio/Sources/Localizations/Main/Localizable.xcstrings

```diff
@@ -86472,7 +86472,7 @@
         }
       }
     },
-    "Enable Fatty Meal Option" : {
+    "Enable Reduced Bolus Option" : {
```

Comment:
@t1dude can you help review the Localization changes? We should have these strings re-translated since we're updating the text. I think these are right, but I'm not 100% sure how Crowdin works.